tests/integration/ha_tests/helpers.py

```diff
@@ -450,11 +423,21 @@ async def reused_storage(ops_test: OpsTest, unit_name, removal_time) -> bool:
 
         item = json.loads(line)
 
+        # We need a msg
```

Comment:
can you add why?

Reply:
Thing is I don't know, we are not using the message, I was just trying to add a bit more documentation ^^"